package.json

```diff
@@ -86,7 +86,7 @@
     "ember-source": "4.2.0",
     "ember-source-channel-url": "^3.0.0",
     "ember-template-lint": "^4.3.0",
-    "ember-try": "^2.0.0",
```

Comment:
reverted due to errors I don't want to debug atm (I believe they're already reported)